onnxruntime/test/util/default_providers.cc

```diff
@@ -209,6 +209,22 @@ std::unique_ptr<IExecutionProvider> DefaultSnpeExecutionProvider() {
 #endif
 }
 
+std::unique_ptr<IExecutionProvider> DefaultQnnExecutionProvider() {
+#if defined(USE_QNN)
+  ProviderOptions provider_options_map;
+  // Limit to CPU backend for now. TODO: Enable HTP emulator
+  std::string backend_path = "./libQnnCpu.so";
+#if defined(_WIN32) or defined(_WIN64)
```

Comment:
I didn't know you could use `or` in preprocessor macros. Should probably just use `||` for consistency.